source/content.ts

```diff
@@ -122,6 +122,7 @@ import './features/update-pr-from-base-branch';
 import './features/hide-disabled-milestone-sorter';
 import './features/tag-changelog-link';
 import './features/link-to-file-in-file-history';
+import './features/deep-blame';
```

Comment:
Maybe this. It's what GitHub calls the button
```suggestion
import './features/deep-reblame';
```